applications/endoscopy_tool_tracking_distributed/cpp/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+	"application": {
+		"name": "Endoscopy Tool Tracking Distributed",
+		"authors": [
+			{
+				"name": "Holoscan Team",
+				"affiliation": "NVIDIA"
+			}
+		],
+		"language": "C++",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "2.0.1",
+			"tested_versions": [
+				"2.0.1"
```

Comment:
The cpp/CMakeLists.txt specifies 2.1.0? Should it be 2.0.1 or should this be 2.1.0?

Reply:
Thanks! Good catch, it should've been 2.1.0.